develop/toc.yml

```diff
@@ -2464,6 +2464,11 @@
         items: 
         - name: '@includepages'
           topicUid: Protocol.SNMP-includepages
+      - name: SystemOptions
```

Comment:
In Protocol.md (develop/schemadoc/Protocol/Protocol.md), add following entry to the Children table:

|&nbsp;&nbsp;[SystemOptions](xref:Protocol.SystemOptions)|[0, 1]|Specifies system related options.|